content/linear_algebra/README.md

```diff
@@ -3,6 +3,11 @@
 Incharge: Sathiya
```

Comment:
I think we should slightly change the contents of the linear algebra session so that it's _not_ very application-oriented. For instance, I'd put point 6 - "Linear system of equations" - at the beginning so that the session really starts and then continues with the basics - linear systems, Gaussian elimination, column/row spaces, bases, things like that.

I also think we don't have to talk about matrix decompositions here since there's a whole MRC lecture on that.